src/service/templates/models/mongoose/templates/service.json

```diff
@@ -1,7 +1,16 @@
 {
-  "require": "feathers-mongoose",
-  "options": {
-    "model": "./service.model",
+  "require": "{{answers.model.require}}",
```

Comment:
Yeah I assumed so. However I haven't yet completely figured out how to manage that process properly since there will be a number of different resources configurations (filters, hooks etc) that also have to be merged into the `service.json`. This was a simple way to make sure that models are generated with sane defaults for now. I just noticed I have to add the default hooks back in for this to be considered full sane defaults.